tests/EventListener/ExceptionListenerTest.php

```diff
@@ -63,7 +63,7 @@ public function testHandleException(): void
         self::assertEquals(Logger::CRITICAL, $record['level']);
         self::assertEquals($exception, $record['context']['exception']);
         self::assertEquals([], $record['context']['context']);
-        self::assertStringStartsWith('exception-', $record['context']['exceptionId']);
+        self::assertSame($responseBody['exceptionId'], $record['context']['exceptionId']);
```

Comment:
no budiž, ale v tom případě bych to přesunul k řádku 57, protože pak patří ty dva asserty dohromady

Reply:
Vidis, prehlidl jsem, ze se tam vys taky kontroluje format exception ID. No aby to nebylo takove polovicate, bylo by teda lepsi:
* nechat tam oba checky na format a potom nakonec porovnat ty hodnoty
* checky na format uplne vyhodit a necht tam jen check na rovnost hodnot

Osobne bych tam klidne nechal i ty checky na format - cim vic checku, tim vic adidas, ale mozna tam uvidis nejakou zbytecnost?

Reply:
check na rovnost hodnot je sám o sobě je o ničem, protože empty string = empty string a seš bez exception ID. Takže ne zbytečný není ani jeden, nechal bych je tam check na formát a check na rovnost, ale dal bych je vedle sebe, abys to příště nepřehlídl